spec/decorators/news_tematica/news_tematica_decorator_spec.rb

```diff
@@ -1,6 +1,77 @@
-require 'spec_helper'
+require 'rails_helper'
 
 describe NewsTematica::NewsTematicaDecorator do
+  describe 'locale_para_enlaces' do
+    it 'should return :es when dominio_de_envio is blank' do
+      expect(build(:news_tematica, dominio_de_envio: nil).decorate.locale_para_enlaces).to eq(:es)
+      expect(build(:news_tematica, dominio_de_envio: '').decorate.locale_para_enlaces).to eq(:es)
+    end
+
+    it 'should return :mx when dominio_de_envio is "mx"' do
+      expect(build(:news_tematica, dominio_de_envio: 'mx').decorate.locale_para_enlaces).to eq(:mx)
+    end
+  end
+
+  describe 'remitente' do
+    it 'should return info@midominio.com when dominio_de_envio is :es' do
+      expect(build(:news_tematica, dominio_de_envio: 'es').decorate.remitente).to eq('info@midominio.com')
+    end
+
+    it 'should return info.mx@midominio.com when dominio_de_envio is :mx' do
+      expect(build(:news_tematica, dominio_de_envio: 'mx').decorate.remitente).to eq('info.mx@midominio.com')
+    end
+
+    it 'should return info.xx@midominio.com when dominio_de_envio is :xx' do
+      expect(build(:news_tematica, dominio_de_envio: 'xx').decorate.remitente).to eq('info.xx@midominio.com')
+    end
+  end
+
+  describe 'titulares' do
+    let!(:titular_mx) { create(:titular, mx: true, es: false, fecha_titulares: 1.day.ago) }
+    let!(:titular_es) { create(:titular, mx: false, es: true, fecha_titulares: 1.day.ago) }
+    let!(:titular_global) { create(:titular, mx: true, es: true, fecha_titulares: 1.day.ago) }
+    let(:news) { create(:news_tematica, fecha_desde: 1.week.ago, fecha_hasta: Time.now).decorate }
+
+    context 'en una news mexicana' do
+      before { allow(news).to receive(:locale_para_enlaces) { :mx } }
+
+      it 'devuelve titulares visibles en mx' do
+        expect(news.titulares.map(&:id)).to match_array([titular_mx.id, titular_global.id])
+      end
+    end
+
+    context 'en una news global o española' do
+      before { allow(news).to receive(:locale_para_enlaces) { :es } }
+
+      it 'devuelve titulares visibles en es' do
+        expect(news.titulares.map(&:id)).to match_array([titular_es.id, titular_global.id])
+      end
+    end
+  end
+
+  describe 'temas' do
+    let!(:tema_mx) { create(:tema, mx: true, es: false, created_at: 1.day.ago) }
+    let!(:tema_es) { create(:tema, mx: false, es: true, created_at: 1.day.ago) }
+    let!(:tema_global) { create(:tema, mx: true, es: true, created_at: 1.day.ago) }
+    let(:news) { create(:news_tematica, fecha_desde: 1.week.ago, fecha_hasta: Time.now).decorate }
```

Comment:
Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.now.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.to_i`, `Time.now.to_f` instead.